### ISSUE TITLE
Batch size correction to val loss in prediction module

### ISSUE THREAD

**Issue by jlevy44** [Christensen-Lab-Dartmouth/MethylNet#5]
Hyperparameter scan should take into account batch size for calculation of val loss, val loss should not be dependent on number samples
